bin/pre-push.sh

```diff
@@ -31,8 +31,9 @@ if [ $? -ne 0 ]; then
 	exit 1
 fi
 
-# Ensure both branches are tracked or check-changelogger-use will fail.
-git checkout $PROTECTED_BRANCH --quiet
-git checkout $CURRENT_BRANCH --quiet
+# Ensure both branches are tracked or check-changelogger-use will fail. Note we pass hooksPath
+# to avoid running the pre-commit hook.
+git -c core.hooksPath=/dev/null checkout $PROTECTED_BRANCH --quiet
```

Comment:
This is required to stop these 2 checkouts triggering post-checkout. I tried HUSKY=0 but it doesn't work from inside another hook.